.github/workflows/publish-r2.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: Publish to Cloudflare R2
+
+on:
+  workflow_call:
+    inputs:
+      path:
+        description: Path to the directory to publish (defaults to "public/").
+        type: string
+        default: "public/"
+      lfs:
+        description: Whether to clone with git-lfs (defaults to false).
+        type: boolean
+        default: false
+
+    secrets:
+      R2_ACCESS_KEY_ID:
+        required: true
+      R2_SECRET_ACCESS_KEY:
+        required: true
+      R2_ACCOUNT_ID:
+        required: true
+      R2_BUCKET:
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    container: debian:bookworm
+    steps:
+      - name: Install dependencies
+        run: |
+          apt-get update
+          apt-get install --yes git git-lfs rclone
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          lfs: ${{ inputs.lfs }}
+      - name: Sync to R2
+        env:
+          RCLONE_S3_PROVIDER: "Cloudflare"
+          RCLONE_S3_ACCESS_KEY_ID: ${{ secrets.R2_ACCESS_KEY_ID }}
+          RCLONE_S3_SECRET_ACCESS_KEY: ${{ secrets.R2_SECRET_ACCESS_KEY }}
+          RCLONE_S3_ENDPOINT: "https://${{ secrets.R2_ACCOUNT_ID }}.r2.cloudflarestorage.com"
```

Comment:
This line is apparently too long - can we bump the limit to like 100 characters instead of 80? I guess I could break it down, but that would decrease readability IMO.

Reply:
We can either bump to 100 or just ignore this one instance by adding `# yamllint disable-line rule:line-length`, whichever you think is easier. I don't feel strongly either way.

Reply:
I'll do the disable-line for now - once someone adds a second too-long-line we can bump the limit :)